tools/pydocs/src/conf.py

```diff
@@ -0,0 +1,132 @@
+# This file is execfile()d with the current directory set to its
```

Comment:
Before you merge, it would be great to add a link to the original version this is based on.